lambda/logging.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_cloudwatch_log_group" "lambda" {
-  name              = "/lambda/${var.function_name}"
+  name              = "/aws/lambda/${var.function_name}"
```

Comment:
Does this mean we use the default log group? Do we even need to create it then, or can we import it with a `data` instruction after the lambda was created?

Reply:
If we do not create it, AWS will also create the log group with the same name. However, we need to reimport with the `data` instruction.

So I think it is better if we create it so we can save the efforts to import the log group

Reply:
Ok. Both approaches would work I guess. I have no strong opinion on which one is better.